static/js/main.js

```diff
@@ -191,20 +196,50 @@ var ep = window['ep'] || {
         });
     },
 
-    _updateModuleLockStatus: function(slot_id) {
+    _updateModuleLockStatus: function(slot_id, used_by) {
         if (slot_id == ep._broadcast_slot) {
             // TODO: log warning
         }
         else {
-            $.get(ep.base_url+"/api/v1/lock_module?format=json&slot=" + slot_id,
-                function (username) {
-                    ep._markUsedBy(slot_id, username)
-                });
+            ep._markUsedBy(slot_id, used_by);
         }
     },
 
-    _markUsedBy: function(slot_id, username) {
-        // TODO: add GUI
+    _markUsedBy: function(slot_id, used_by) {
+        var lock_container = $("#module_header_"+slot_id).find(".module_lock");
+        if (used_by) {
+            lock_container.text(used_by);
+            $("#module_header_"+slot_id).toggleClass("open", false);
+        } else {
+            //Module is not locked by anyone
+            lock_container.empty();
+        }
+    },
+
+    _parseWSMessage: function (message) {
+        console.log("Message from ws: " + message);
+        var json = JSON.parse(message);
+        switch (json.msg_type) {
+            case 'MODULE_UPDATE':
+                //Request to update Module info has been received
+                ep._updateModuleUI(json.slot, json.module_name);
+                break;
+
+            case 'LOCK_UPDATE':
+                //Update lock status of the module
+                ep._updateModuleLockStatus(json.slot, json.used_by);
+                break;
+
+            case 'DATA_UPDATE':
```

Comment:
I would prefer to do it in a separate task, as it will require additional testing done